inspirehep/modules/workflows/tasks/refextract.py

```diff
@@ -86,49 +86,43 @@ def extract_journal_info(obj, eng):
 
 def extract_references(filepath):
     """Extract references from PDF and return in INSPIRE format."""
+
+    def field_builder(builder, value_l):
+        for value in value_l:
+            builder(value)
```

Comment:
Now this higher order function is just `map`: you get a function and a list, and you apply the function to each element of the list.